components/project-card/meta/creators.jsx

```diff
@@ -6,7 +6,7 @@ import classNames from 'classnames';
 const Creators = (props) => {
   if (props.creators.length === 0) return null;
 
-  let classnames = classNames(`creator d-block open-sans`, props.className);
+  let classnames = classNames(`my-1 body-small creator d-block`, props.className);
```

Comment:
now that this is for `<p>` we can remove `d-block` as by default `<p>` is block-level element already.

Reply:
Oooo thanks for pointing that out. 👐